doc/radosgw/config-ref.rst

```diff
@@ -664,6 +664,9 @@ Swift Settings
               this option to trust the ``Forwarded`` and ``X-Forwarded-Proto`` headers
               sent by the proxy when determining whether the connection is secure.
               This is required for some features, such as server side encryption.
+              (Never enable this setting if you do not have a trusted proxy in front of
+              radosgw, or else malicious users will be able to set these headers in
+              any request.)
```

Comment:
this description is taken directly from https://github.com/ceph/ceph/blob/1c3dd60/src/common/options.cc#L6671-L6675. can we please update both?

Reply:
Thanks. I've copied and pasted this to `src/common/options.cc`.